Data/DynamicInputPack.cs

```diff
@@ -206,6 +206,17 @@ public ICommand EditTextureCommand
 
         private void EditTexture(DynamicInputTexture texture)
         {
+            //Automatically select suitable zoom.
+            if (texture.ScaleFactor == 1)
```

Comment:
I wonder if this should be a function on the texture.  The program this was patterned after (Photoshop) has a zoom to fit button which we could add later.

Reply:
i will work towards this. after the import function, changes are now also live ;)
Smooth zoom values was also adjusted.

Reply:
it seems that I have made a mistake, I will look tomorrow where the error is.

Reply:
should be all ok now :)

Reply:
I admit it's a bit strange.
this was the easiest metode for me to get the length.